README.md

```diff
@@ -5,6 +5,14 @@ iOS and tvOS SDK for [Treasure Data](http://www.treasuredata.com/). With this SD
 
 Also, there is an alternative SDK written in Swift [https://github.com/recruit-lifestyle/TreasureDataSDK](https://github.com/recruit-lifestyle/TreasureDataSDK). Note, however, that it does not support current GDPR functionality in the mainstream TD SDKs.
 
+## Migration to version 1
+
+Version 1 has major changes that are not backward compatible with previous versions. If you are upgrading from version 0.9.0 or earlier, your code will not run correctly without doing these following steps:
+- API endpoint has changed to Ingestion Endpoint. For now, the only available endpoint is https://us01.records.in.treasuredata.com.
```

Comment:
Just a notes that we need to check with BE ingest whether this still valid , or team had provided support in all sites

Reply:
Thanks for spotting this. Please check the latest commit for more general wording.